.gitignore

```diff
@@ -55,3 +58,6 @@ tesk_application.conf
 **/venv/
 exome_germline_single_sample_v1.3/
 **/*.pyc
+
+# GHA credentials
+gha-creds-*.json
```

Comment:
Is this and `bloop` being used anywhere? 🤔

Reply:
I think some folks on the team use bloop for Scala builds. I assume gha-creds is for being able to test things locally?

Reply:
Not used in this PR directly, I assume Justin added these during his development work.